src/app/containers/PageHandlers/withPageWrapper.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PageWrapper from '../Layouts';
+
+const WithPageWrapper = Component => {
+  return function WithPageWrapperComponent(props) {
+    return (
+      <PageWrapper {...props}>
```

Comment:
With pageWrapper you would be able to pass the in the Page Layout that you would like. So we could have a different layout for topic pages for instance.